src/CSET/operators/constraints.py

```diff
@@ -62,6 +62,26 @@ def generate_var_constraint(varname: str, **kwargs) -> iris.Constraint:
     return varname_constraint
 
 
+def generate_model_level_constraint(model_level_name: str, **kwargs) -> iris.Constraint:
```

Comment:
```suggestion
def generate_model_level_constraint(model_level_number: str, **kwargs) -> iris.Constraint:
```
Use more descriptive parameter name. If model levels are always only refered to by number I wonder if we should just have this as `model_level`.

Reply:
chose model_level_number as sometimes users might otherwise confuse it with model_level_height